site/public/js/simple-grading.js

```diff
@@ -667,34 +670,34 @@ function setupSimpleGrading(action) {
         }
     }
 
-    // default key movement
-    $(document).on("keydown", function(event) {
-        // if input cell selected, use this to check if cursor is in the right place
-        var input_cell = $("input.cell-grade:focus");
-
-        // if there is no selection OR there is a selection to the far left with 0 length
-        if(event.code === "ArrowLeft" && (!input_cell.length || (
-                input_cell[0].selectionStart == 0 &&
-                input_cell[0].selectionEnd - input_cell[0].selectionStart == 0))) {
-            event.preventDefault();
-            movement("left");
-        }
-        else if(event.code === "ArrowUp") {
-            event.preventDefault();
-            movement("up");
-        }
-        // if there is no selection OR there is a selection to the far right with 0 length
-        else if(event.code === "ArrowRight" && (!input_cell.length || (
-                input_cell[0].selectionEnd == input_cell[0].value.length &&
-                input_cell[0].selectionEnd - input_cell[0].selectionStart == 0))) {
-            event.preventDefault();
-            movement("right");
-        }
-        else if(event.code === "ArrowDown") {
-            event.preventDefault();
-            movement("down");
-        }
-    });
+    // // default key movement
```

Comment:
If this isn't needed anymore it might be better to just remove it

Reply:
Ended up readding this, realized there's a minor bug with arrows keys on checkpoint gradeables.